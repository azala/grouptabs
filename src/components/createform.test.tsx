import React from "react";
import { render } from "@testing-library/react";
import CreateForm from "./createform";

it("renders form", () => {
  const { container } = render(
    <CreateForm
      tabName="Badminton"
      onTabNameChange={jest.fn()}
      onSubmit={jest.fn()}
    />
  );
  expect(container.firstChild).toMatchSnapshot();
});
