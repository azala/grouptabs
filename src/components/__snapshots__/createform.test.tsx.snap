// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders form 1`] = `
<form
  class="create-form"
>
  <input
    class="full-width"
    placeholder="Tab name â€¦"
    type="text"
    value="Badminton"
  />
  <button
    class="create"
  >
    Create
  </button>
</form>
`;
